bricks/basic-bricks/dist-types/popover-container/index.d.ts

```
@@ -0,0 +1,87 @@
+import type { MenuIcon, UseBrickConf } from "@next-core/brick-types";
+import type { TooltipPlacement } from "antd/lib/tooltip";
+import type { ActionType, AlignType } from "rc-trigger/lib/interface";
+import type { MenuIcon } from "@next-core/brick-types";
```

Biome (2.4.4)

[error] Shouldn't redeclare 'MenuIcon'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)